### DISCUSSION TITLE
An all in one Bot

### DISCUSSION THREAD

**Discussion by JustFinchy** [o3de/sig-operations#14] [Ideas]
Currently there are multiple bots on the server. 

There needs to be reduced to have a custom bot solution to have everything we need. Its fine with 1 or 2 bots, but anything more starts to clutter the server.

**Comment by OBWANDO** [comment]
I am not sure I am tracking here. Having 2, 3 or even 5 bots pales in contrast to the 1200 people on the server, and they are buried in the list.
Consolidating bots can be done, but that is time and effort, and right now with how young the community is, communication and functionality are more important.

**Reply by JustFinchy** [reply]
Im more speaking at a later date. I usually dont trust having a lot as they can go down anytime and cause issues. seen it happen again and again.
